app/app.go

```diff
@@ -336,6 +342,16 @@ func New(
 		appCodec, keys[govtypes.StoreKey], app.GetSubspace(govtypes.ModuleName), app.AccountKeeper, app.BankKeeper,
 		&stakingKeeper, govRouter,
 	)
+
+	app.NftKeeper = nftmodulekeeper.NewKeeper(
+		keys[nftmodulekeeper.StoreKey],
+		appCodec,
+		app.AccountKeeper,
+		//keys[nftmoduletypes.MemStoreKey],
+		app.BankKeeper,
+	)
+	nftModule := nftmodule.NewAppModule(appCodec, app.NftKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry)
```

Comment:
These came semi-filled in from the starport scaffold, but with errors, so I modeled it after the current version in the [sdk](https://github.com/pomifer/cosmos-sdk/blob/151c6e85e6e323e54519135a94ca6ab2abce0fa4/simapp/app.go#L325).